src/lib/stitching/vortex/link.ts

```diff
@@ -35,10 +34,14 @@ export const createVortexLink = () => {
           }
         })
       }
+
+      const token = graphqlContext.appToken || VORTEX_TOKEN
+      const bearer = `Bearer ${token}`
+
       return {
         headers: {
           ...headers,
-          Authorization: `Bearer ${VORTEX_TOKEN}`,
+          Authorization: bearer,
```

Comment:
This approach looks good! the graphql context is exactly how one passes this kind of data around.